app/Http/Livewire/StreamList.php

```diff
@@ -20,14 +19,12 @@ public function render(): View
             ->when($this->isArchive, function($builder) {
                 $builder->finished()->fromLatestToOldest();
             }, function($builder) {
-                $builder->upcoming()->orderBy('scheduled_start_time');
+                $builder->upcoming()->fromOldestToLatest();
             })
             ->paginate(10);
 
         return view('livewire.stream-list', [
-            'streamsByDate' => $streams->setCollection(
-                (new SortStreamsByDateAction())->handle($streams->getCollection())
-            ),
+            'streamsByDate' => $streams,
```

Comment:
@christophrumpel I moved the `(new SortStreamsByDateAction())->handle` to the view. I don't like how i reference that in the view. But i have done that because of the wrong `Showing 1 to 1 of 100 results`.

Reply:
Can you tell me again what issues we have and what you mean by "Showing 1 to 1 of 100 results"?

Reply:
On the live site, on the archive you see: 
![image](https://user-images.githubusercontent.com/477362/125430805-3e7a8bbc-a4b7-4da9-8f24-02d6740f28d9.png)
(Because the text was the same as the background i didn't see it earlier). 

It says `1 to 6`. But it should be `1 to 10` on this page (10 per page). 
It count the groups instead of the items.

Reply:
Ah right, didn't see that.

Yeah, so I agree that we don't want the action being used inside the view. What about we just delete the `div` telling what we currently see? Because the rest works, right?

Reply:
It looks like the rest still works if we overwrite the collection. So removing the div is indeed a solution.